.pipelines/ci.yml

```diff
@@ -417,25 +429,14 @@ stages:
         call activate pyenv
         python -m pip install --upgrade pip
         python -m pip install onnxruntime==$(ort.version)
-        python -m pip install -r requirements-dev.txt
-      displayName: Install requirements{-dev}.txt and cmake python modules
+      displayName: Install requirements and cmake python modules
 
     - script: |
         call activate pyenv
         set CMAKE_ARGS=-DOCOS_ONNXRUNTIME_VERSION=$(ort.version)
         python -m pip install .
       displayName: Build the wheel
 
-    - script: |
```

Comment:
will all python version skip the unit test, 
you can try to   "condition: neq(variables['python.version'], '3.12')" to only skip if python version is 3.12

Reply:
ah of course, sorry - fixed.